alpine-ruby/sbin/strict-mode.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Reference:
+# http://www.davidpashley.com/articles/writing-robust-shell-scripts/
+# http://kvz.io/blog/2013/11/21/bash-best-practices/
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
+set -o errexit    # Exit when an expression fails
+set -o pipefail   # Exit when a command in a pipeline fails
+set -o nounset    # Exit when an undefined variable is used
+set -o noglob     # Disable shell globbing
+set -o noclobber  # Disable automatic file overwriting
```

Comment:
While I like the verbose explanations I dislike 5 calls to `set`. What about using the following format:

```
set \
  - o errexit \ # documentation
  - o other_flag \ # more docs
```

?